### DISCUSSION TITLE
How to insert a tab spaces at the end of line?

### DISCUSSION THREAD

**Discussion by sgon00** [helix-editor/helix#9930] [Q&A]
Hi, I don't want to type four times spaces at the end of line to insert four spaces. Why does Tab button jump to the end of file in insert mode instead of inserting spaces? I don't really get it.

I can insert spaces at any places with Tab key except at the end of line in the Insert mode.  How can I do that? 

Thanks a lot.

**Comment by pascalkuthe** [comment]
That is the smart tab functionality. You can either remove the mapping in your config (test the documentation it's extension about smart tab) or you can use c-tab

**Reply by sgon00** [reply]
By `c-tab`, do you mean `Shift+Tab`? I tried `Shift+Tab` and it works. `Ctrl+Tab` does NOT work. Thanks a lot.
